pyproject.toml

```diff
@@ -1,6 +1,37 @@
+[build-system]
+requires = ["setuptools>=61.2", "setuptools_scm[toml]>=3.4.3"]
+build-backend = "setuptools.build_meta"
+
 [project]
 name = "ntia_conformance_checker"
-version = "0.0.1"
-dependencies = [
-    "spdx-tools>=0.7.0rc0",
+version = "0.2.0"
+authors = [{name = "John Speed Meyers", email = "TBD@gmail.com"}]
+maintainers = [
+    {name = "John Speed Meyers", email = "TBD@gmail.com"},
+    {name = "SPDX group at the Linux Foundation and others"},
```

Comment:
```suggestion
maintainers = [
    {name = "John Speed Meyers", email = "johnmeyersster@gmail.com"},
    {name = "Gary O'Neall", email = "gary@sourceauditor.com"},
    {name = "Josh Lin", email = "linynjosh@gmail.com"},
    {name = "SPDX group at the Linux Foundation and others"},
```

cc @goneall @linynjosh - I figured that _at least_ you both should also be in this list of maintainers. Please say something if you think otherwise :)